### PR TITLE
bug fix label outline(int to float)

### PR DESCRIPTION
**Pull request by newnon**
fix label outline

### CODE REVIEW

File: cocos/2d/CCLabel.cpp [hunk 1]
```diff
@@ -1056,11 +1056,11 @@ void Label::enableGlow(const Color4B& glowColor)
     }
 }
 
-void Label::enableOutline(const Color4B& outlineColor,int outlineSize /* = -1 */)
+void Label::enableOutline(const Color4B& outlineColor,float outlineSize /* = -1 */)
```
Comment (dumganhar) [inline cocos/2d/CCLabel.cpp:1059]: Hi @newnon, did you get any reason that outlineSize should be `float` instead of `int`?

**Comment by newnon** [comment]
because in FontFreeType field  float _outlineSize and it works well sometime you need float outline it's normal

```
FT_Stroker_Set(_stroker,
            (int)(_outlineSize * 64),
            FT_STROKER_LINECAP_ROUND,
            FT_STROKER_LINEJOIN_ROUND,
            0);
```

**Comment by minggo** [comment]
How to reproduce the issue? Could you please add a test case and paste expected effect and current wrong effect?

**Comment by minggo** [comment]
there is conflict in this PR, please send a new one, thanks

**Pull request closed without merging**